SonarQube.Bootstrapper/App.config

```diff
@@ -17,10 +17,10 @@
         <value>SonarQube.MSBuild.PostProcessor.exe</value>
       </setting>
       <setting name="PreProcessorTimeoutInMs" serializeAs="String">
-        <value>300000</value>
+        <value>-1</value>
```

Comment:
According to MSDN: 

-1 = wait forever
0 = don't wait at all, just return true / false if the process has existed or not
1+ = wait for the specified number of ms

I'll add a comment to the BootstrapperSettings about this.